leetcodePython/SlidingWindow/slidingwindow_1839.py

```diff
@@ -0,0 +1,18 @@
+class Solution:
+    def longestBeautifulSubstring(self, word: str) -> int:
+        vowels = ['a', 'e', 'i', 'o', 'u']
+        prev_ch = 'z'
+        vowel_count = 0
+        left = 0
+        longest = 0
+        for right in range(len(word)):
+            if word[right] > prev_ch:
+                vowel_count += 1
+            elif word[right] < prev_ch:
+                vowel_count = 1
```

Comment:
Even if the current character is not `a` we dont care. Since if it is not `a` it will not be able to reach all vowels requirements. 

But this is important for situation when character is `a`. Then `vowel_count` must be 1